src/Expression/ExprBuilder.cpp

```diff
@@ -496,8 +496,8 @@ Value* ExprBuilder::evalExpr(const FileContent* fC, NodeId parent,
         break;
       }
       case VObjectType::slRealConst: {
-        const std::string& real = fC->SymName(child);
-        std::istringstream os(real);
+        const std::string_view real = fC->SymName(child);
+        std::istringstream os(real.data());
```

Comment:
This is not a valid use of `data()`
Since this is Why not use the new `parseDouble()` function here ?

Reply:
Sent #3392